### ISSUE TITLE
I think I have it

### ISSUE THREAD

**Issue by rajesh06** [rajesh06/The-Value-of-Insurance#3]
Hi Zhenkai - Please take a look, I think we have it. If you agree, we only need to write a conclusion. (and do some final read-throughs)

**Comment by zhenkaizhu** [comment]
Yes, I agree with the new formula, so sorry I miss these 2 issue messages again without an email reminder.
